Assets/Scripts/Controllers/TileSpriteController.cs

```diff
@@ -56,11 +56,16 @@ protected override void OnCreated(Tile tile)
         sr.sprite = SpriteManager.current.GetSprite("Tile", "Empty");
         sr.sortingLayerName = "Tiles";
 
-        OnChanged(tile);
+        OnChanged(tile,true);
+    }
+
+    protected override void OnChanged(Tile obj)
+    {
+        OnChanged(obj, false);
     }
 
     // This function should be called automatically whenever a tile's data gets changed.
-    protected override void OnChanged(Tile tile)
+    protected void OnChanged(Tile tile, bool forceTile)
```

Comment:
As far as I can tell, forceTile is never used and I can't figure out what it means.